datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -23,6 +25,36 @@ def __is_canonical(version):
     return re.match(P, version) is not None
 
 
+def __find_shipped_integrations():
+    # Recurse up from site-packages until we find the Agent root directory.
+    # The relative path differs between operating systems.
+    root = os.path.dirname(os.path.abspath(__file__))
+    filename = 'requirements-agent-release.txt'
+
+    integrations = set()
+
+    while True:
+        file_path = os.path.join(root, filename)
+        if os.path.isfile(file_path):
+            break
+
+        new_root = os.path.dirname(root)
+        if new_root == root:
+            return integrations
```

Comment:
Wouldn't this return an empty set? Why would it ever be empty?

Reply:
CI maybe

Reply:
I see, thanks for explaining. I guess we just have to be careful not to return an empty set on a user machine.